recipes/openms/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+
+# Otherwise libraries won't be found during linking on Linux
+export LIBRARY_PATH=${PREFIX}/lib
+export LD_LIBRARY_PATH=${PREFIX}/lib
+
+# Otherwise Error: dyld: Symbol not found: _iconv Referenced from: /usr/lib/libarchive.2.dylib
+# Expected in: /Users/runner/mambaforge/conda-bld/openms-meta_1686666744655/_h_env_placehold_/lib/libiconv.2.dylib
+#export DYLD_LIBRARY_PATH=${PREFIX}/lib
```

Comment:
```suggestion
```

can we try removing these?

Reply:
We can. I think I did before but we can have a look at the errors together.